presidio-anonymizer/presidio_anonymizer/entities/engine/anonymizer_result.py

```diff
@@ -0,0 +1,57 @@
+import logging
+from typing import Dict
+
+from presidio_anonymizer.entities.engine import PIIEntity
+from presidio_anonymizer.entities.engine.result import OperatorResult
+
+
+class AnonymizerResult(PIIEntity):
+    """Information about the encrypted entity."""
+
+    def __init__(self, start: int,
+                 end: int, entity_type: str) -> 'AnonymizerResult':
+        """
+        Have the information about the entity we already encrypted and want to decrypt.
+
+        :param key: the key used to anonymize/encrypt the text.
+        :param start: start index in the anonymized text.
+        :param end: end index in the anonymized text.
+        """
+        PIIEntity.__init__(self, start, end, entity_type)
+        self.logger = logging.getLogger("presidio-anonymizer")
+
+    def __gt__(self, other) -> bool:
```

Comment:
why is this different than PIIEntity? other than the difference in order, what does this provide that the base doesn't?

Reply:
Correct. Removed it, and moved it to the PIIEntity.